mw-webapp/src/pages/authPage/AuthPage.tsx

```diff
@@ -4,12 +4,12 @@ import styles from "src/pages/authPage/AuthPage.module.scss";
 export const AuthPage = () => {
   return (
     <div className={styles.container}>
-      <Link to={"/"}>
+      <Link to={"sign-in"}>
```

Comment:
Keep it simple, you don't need 3 pages for auth

Reply:
Ok. I will use only auth with Google account and delete sign in and sign up pages

Reply:
Done